plugin.json

```
@@ -2,6 +2,10 @@
     "id": "com.mattermost.plugin-starter-template",
     "name": "Plugin Starter Template",
     "description": "This plugin serves as a starting point for writing a Mattermost plugin.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-starter-template",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-starter-template/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-starter-template/releases/tag/vX.X.X",
```

Should we preemptively fill that out?
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-starter-template/releases/tag/v0.1.0",
```